app/views/chats/_chat.html.haml

```diff
@@ -0,0 +1,23 @@
+.home-left-chat-section
+  .chat-top
+    .back-button
+      <
+    .h4 Chat
+  .chat-middle{ data: { message_path: messages_path, user_id: current_user.id, chat_id: match.chat.id } }
+    .topic-info
+      .left-topic-info
+        = image_tag "#{match.first_target.topic.title}x2.png", class: 'topic-info-img'
+      .right-topic-info
+        - if match.first_target.user_id == current_user.id
```

Comment:
This logic shouldn't be in the partial. Move it to a method on the `Match` model. For example, you can do:
```
def other_user_target(user)
  first_target.user == user ? second_target : first_target
end
```
Send this `target` as a local param